mirdata/datasets/haydn_op20.py

```diff
@@ -0,0 +1,309 @@
+"""haydn op20 Dataset Loader
+
+..
+    This dataset has a relative dependency with Music21.
+    .. admonition:: Dataset Info :class: dropdown
+        This dataset accompanies the Master Thesis from Nestor Napoles. It is a manually-annotated corpus of harmonic
+        analysis in harm syntax.
+
+        The dataset contains the following scores:
+        Haydn, Joseph
```

Comment:
Docs are not rendering at all

Reply:
Dale!

Reply:
![Captura de pantalla 2021-03-07 a las 14 09 28](https://user-images.githubusercontent.com/16807496/110241009-e587f000-7f4e-11eb-93de-76aa128615a9.png)